source/integrate-with-integration-environment/integrate-with-code-flow.html.md.erb

```diff
@@ -223,7 +223,7 @@ Location: https://YOUR_REDIRECT_URI?error=invalid_request
   </tr>
   <tr>
     <td class="tg-0pky"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000;background-color:transparent">access_denied</span></td>
-    <td class="tg-0pky"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000;background-color:transparent">The identity evidence provided by your user has a lower score than the identity confidence specified in your request. As a result, GOV.UK One Login could not return the medium level of identity confidence (<code>Cl.Cm.P2</code>) and instead returned a lower level of identity confidence. <br></br>To resolve this, make sure your user has entered the correct information for their identity evidence. If they have entered it correctly and this error is still appearing, you must ask your user to provide different identity evidence which meets a medium level of identity confidence.</span></td>
+    <td class="tg-0pky"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000;background-color:transparent">The identity evidence provided by your user has a lower score than the identity confidence specified in your request. As a result, GOV.UK One Login could not return the medium level of identity confidence (<code>Cl.Cm.P2</code>) and instead returned a lower level of identity confidence. <br></br>This could also happen if your user selected **Cancel** during the identity verification process.</span></td>
```

Comment:
I think the code should be <code>P2</code>

Should we also mention the lost session as a possible access_denied